java-symbol-solver-core/src/test/resources/MethodCalls.java.txt

```diff
@@ -1,29 +1,51 @@
+import java.util.List;
+
 class MethodCalls {
 
-	public int member;
-	
-	public MethodCalls getSelf()
-	{
-		return this;
-	}
-
-	public void foo()
-	{
-	}
-	
-	void bar1()
-	{
-		getSelf().getSelf().foo();
-	}
-
-	int bar2()
-	{
-		return getSelf().m;
-	}
-	
-	void varidicMethod(String... s)
-	{
-		this.varidicMethod("test");
-	}
+    public int member;
+    
+    public MethodCalls getSelf()
+    {
+        return this;
+    }
+
+
+    public void foo()
+    {
+    }
+
+    void bar1()
+    {
+        getSelf().getSelf().foo();
+    }
+
+    int bar2()
+    {
+        return getSelf().m;
+    }
+
+    void inheritedInterfaceMethod(){
+        List<Integer> list;
+        list.toString();
+    }
+
+    void variadicTest(){
+        String[] varArg = new String[2];
+        variadicMethod("a");
+        variadicMethod(varArg);
+    }
+
+    int variadicMethod(String s){
+        return 1;
+    }
+
+    void variadicMethod(String... s){
```

Comment:
I misspelled variadic into varidic but I would avoid having a method with a name so close, it could cause confusion